packages/map-template/src/components/WebComponentWrappers/RouteInstructionsStep/RouteInstructionsStep.jsx

```diff
@@ -102,4 +104,12 @@ const RouteInstructionsStep = forwardRef(({ totalSteps, activeStep, previous, or
     </mi-route-instructions-step>
 });
 
+RouteInstructionsStep.propTypes = {
+    totalSteps: PropTypes.array, // <- TODO There is a discrepancy between the JSDoc and the PropTypes
```

Comment:
I think that all the "object" JSDoc params is just the result of a lazy auto-complete at some point. Just update the JSDoc with what is correct.